### PR TITLE
 Restaurant Page

### PR DESCRIPTION
**Pull request by Emmyn5600**
#### . Added the Contact us section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
-# Restaurant_Page
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# restaurant-page
+
+> In this project, I built a simple website for a restaurant.
```
Comment (bellom) [inline README.md:5]: Your README file is missing live demo link.
Kindly add it

File: README.md [hunk 2]
```diff
@@ -1 +1,46 @@
...
+
+## Built With
+
+- HTML
+- CSS3
+- JAVASCRIPT
+
+
+## Getting Started
```
Comment (bellom) [inline README.md:18]: This project came after Webpack lesson, we need to use the knowledge in this project.

According to the pathwright, you need to add scripts that are necessary to build your project in package.json.

image

https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635163/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Emmyn5600  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (GzimAsani) [inline package.json:1]: Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :red_circle: 

### Hello @Emmyn5600, good job improving your readme :+1:  but unfortunately you have not included the script inside your package.json file as the previous reviewer asked

### If you have any questions with regards to the requirements please add a comment here and I will try to reply ASAP.

# HAPPY CODING :computer: :desktop_computer:

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :red_circle: 

### Hello again @Emmyn5600, please implement the requirement that I have asked previously
**Example:**
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },


# HAPPY CODING :computer: :desktop_computer:

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi @Emmyn5600, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**